index.js

```diff
@@ -7,6 +7,8 @@ import normalizePath from 'normalize-path'
  * @typedef RecmaExportFilepathOptions
  * @property {boolean} [absolute=false]
  *   If true, use an absolute path. By default a relative path is used.
+ * @property {string} [cwd]
```

Comment:
Could you fix the type error? I think this should do it.

```suggestion
 * @property {string | undefined} [cwd]
```

Reply:
I actually tried this before you ask. But VS Code still shows error. I read JSDoc specification that `[cwd]` means `cwd` is optional. So I took a different approach to fix the VS Code error.

Reply:
I don’t like this solution, but I believe you tried this is the only thing that works.